### PR TITLE
chore: fix typoes & spelling in MITREid guide

### PR DESCRIPTION
**Pull request by jonas-jonas**
https://ory-community.slack.com/archives/C010F7Z4XM1/p1669755303510639
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Review by vinckr — APPROVED** [review]
LGTM

File: docs/hydra/self-hosted/migrating-from-mitreid.mdx [hunk 1]
```diff
@@ -5,26 +5,26 @@ title: Migrating from MITREid
...
+This guide will try to give you some hints on moving your current MITREid instance to Hydra.
 
-#### Scopes
+# Scopes
 
-Scopes in hydra doesn't have a relational entity, every client defines its own array of scopes.
+Scopes in hydra do not have a relational entity, every client defines its own array of scopes.
 
-#### Scope vs Scp
+# Scope vs Scp
```
Comment (aeneasr) [inline docs/hydra/self-hosted/migrating-from-mitreid.mdx:18]: Please don't use H1 excessively.

https://contentmarketinginstitute.com/articles/how-tags-matter-seo/


/cc @vinckr please check this when reviewing.

Comment (aeneasr) [reply docs/hydra/self-hosted/migrating-from-mitreid.mdx:18]: No need to update this, I'm doing this in the refactor.

Comment (jonas-jonas) [reply docs/hydra/self-hosted/migrating-from-mitreid.mdx:18]: Ah, true. We should have a linter for that. Thanks.

Comment (vinckr) [reply docs/hydra/self-hosted/migrating-from-mitreid.mdx:18]: my bad i missed it in the diff 🙈

**Pull request merged**